op-program/client/interop/consolidate.go

```diff
@@ -15,9 +15,12 @@ import (
 	supervisortypes "github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
 	"github.com/ethereum/go-ethereum/common"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/log"
 )
 
+const messageExpiryTimeSeconds = 15552000 // 180 days
```

Comment:
nit: I wonder if there's somewhere we can put this so its shared between op-program and op-supervisor. Maybe in the rollup types with `rollup.Config`?

Reply:
updated to use the rollup config.